user-docs/pupil-player.md

```diff
@@ -149,15 +149,58 @@ This plugin can be used to overlay the eye video on top of the world video. Note
   + `show` - show or hide eye video overlays.
   + `horiz. and vert. flip` - flip eye videos vertically or horizontally
 
-### Analysis Plugins
+### Data Source Plugins
 
-These plugins are simple unique plugins, that operate on the gaze data for analysis and visualizations.
+Data source plugins provide data of a specific topic -- currently `pupil` and `gaze` positions. Plugins that provide data for a specific topic are complementary to each other, e.g. `Pupil Positions From Recording` and `Offline Pupil Detector` (see below). Complementary plugins cannot be run at the same time.
```

Comment:
Edit: Plugins that provide data for the same topic are exclusive. `Pupil Positions From Recording` and `Offline Pupil Detector` can not be run at the same time.  

@papr complementary as a word choice here is misleading.